src/module/apps/sidebar/chat-log.ts

```diff
@@ -308,8 +306,16 @@ export class ChatLogPF2e extends ChatLog<ChatMessagePF2e> {
                         );
                         return;
                     }
+                    if (!actor) return;
+                    const combatant = await CombatantPF2e.fromActor(actor);
+                    if (!combatant) return;
```

Comment:
```suggestion
```

Reply:
This would always show the notification that the initiative was set even if there is no active encounter.